authors/apps/authentication/serializers.py

```diff
@@ -13,7 +15,10 @@ class RegistrationSerializer(serializers.ModelSerializer):
     password = serializers.CharField(
         max_length=128,
         min_length=8,
-        write_only=True
+        write_only=True,
+        error_messages={
+            "min_length": "Password must be more than {min_length} characters"
```

Comment:
something like ``` "min_length": "Password should be atleast  {min_length} characters"```

Reply:
I think this is the same as the one above